e2e/__tests__/customMatcherStackTrace.test.js

```diff
@@ -23,3 +23,11 @@ test('works with custom matchers', () => {
 
   expect(rest).toMatchSnapshot();
 });
+
+test('custom async matchers', () => {
+  const {stderr} = runJest('custom-matcher-stack-trace', ['async.test.js']);
+
+  const result = extractSummary(stderr);
+
+  expect(result).toMatchSnapshot();
```

Comment:
```suggestion
  expect(result.rest).toMatchSnapshot();
```

we don't care about the summary here, only the error message 🙂

Reply:
Right, I missed that. Thank you! 🙂